LibrarySystem/src/Model/Validators/BookValidator.cs

```diff
@@ -2,61 +2,102 @@
+                "Ensure that the Book object is properly instantiated and contains all required fields before attempting to validate it.");
         }
 
         ValidateTitle(book.Title);
         ValidateAuthor(book.Author);
         ValidateISBN(book.ISBN);
+        ValidateGenreBook(book.Genre);
         ValidatePublicationYear(book.PublicationYear);
     }
 
-    private static void ValidateTitle(string title)
+    private void ValidateTitle(string title)
     {
         if (string.IsNullOrWhiteSpace(title))
         {
-            throw new BookException("Title cannot be null, empty, or whitespace");
+            throw new BookException(
+                "Title cannot be null, empty, or whitespace.",
+                SeverityLevel.Medium,
+                "A book title is mandatory and must contain meaningful text. " +
+                "Ensure that the title field is populated with a valid, non-empty string.");
         }
     }
 
-    private static void ValidateAuthor(string author)
+    private void ValidateAuthor(string author)
     {
         if (string.IsNullOrWhiteSpace(author))
         {
-            throw new BookException("Author cannot be null, empty, or whitespace");
+            throw new BookException(
+                "Author cannot be null, empty, or whitespace.",
+                SeverityLevel.Medium,
+                "The author's name is required for proper identification of the book. " +
+                "Please provide a valid author's name, ensuring that it is not empty or composed solely of whitespace.");
         }
     }
 
-    private static void ValidateISBN(string isbn)
+    private void ValidateISBN(string isbn)
     {
         if (string.IsNullOrWhiteSpace(isbn))
         {
-            throw new BookException("ISBN cannot be null, empty, or whitespace");
+            throw new BookException(
+                "ISBN cannot be null, empty, or whitespace.",
+                SeverityLevel.Medium,
+                "The ISBN is essential for cataloging and uniquely identifying the book. " +
+                "Make sure to input a valid ISBN number, and verify that it is not left blank.");
         }
 
         string isbnPattern = @"^(97(8|9))?\d{9}(\d|X)$";
         if (!Regex.IsMatch(isbn, isbnPattern))
         {
-            throw new BookException("ISBN format is invalid. It should be in the format 978-3-16-148410-0 or similar.");
+            throw new BookException(
+                "ISBN format is invalid. It should be in the format 978-3-16-148410-0 or similar.",
+                SeverityLevel.High,
+                "This error suggests that the ISBN entered does not match the standard format. " +
+                "Please verify the ISBN and ensure it conforms to the standard format, such as '978-3-16-148410-0'.");
         }
     }
 
-    private static void ValidatePublicationYear(int year)
+    private void ValidateGenreBook(string genre)
```

Comment:
since you improved the validations, can you improve the validation for the book genre to avoid having books with genres like this?
![Screenshot from 2024-08-22 14-58-25](https://github.com/user-attachments/assets/cdee5823-8579-4132-9e99-4901c714322a)